.gitignore

```diff
@@ -1 +1,2 @@
 */.vscode/*
+.vscode/*
```

Comment:
nit, obviously cause this works more than fine. In the future, if you wanna exclude the `.vscode/` folder from every level, iirc you have you exclude it in the gitignore as `**/.vscode/`

Reply:
ah gotcha, thanks for the tip